tests/test_ci_self_hosted_guard.sh

```
@@ -678,35 +678,44 @@ EOF
+    '"$CREATE_DMG_TOOL" --no-code-sign "$APP_PATH" "$DMG_TMP_DIR"' \
+    '"$CODESIGN_TOOL" --force --timestamp --keychain build.keychain' \
+    '--sign "$APPLE_SIGNING_IDENTITY"' \
+    '"$CODESIGN_TOOL" --verify --verbose=2 "$DMG_RELEASE"' \
+    '"$XCRUN_TOOL" notarytool submit "$DMG_RELEASE"'; do
+    if ! grep -Fq -- "$needle" "$nightly_helper"; then
+      echo "FAIL: nightly notarization helper must sign the DMG through build.keychain before submission: $needle"
       exit 1
     fi
   done
 
+  if grep -Fq -- '--identity="$APPLE_SIGNING_IDENTITY"' "$release_workflow"; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)